README.md

```diff
@@ -0,0 +1,203 @@
+
+### input format
+```json
+{ "statement": {statementBlob}, "references": [ {referenceBlob}... ] }
+```
+
+## SS1: Find good external ID Properties<a name="ss-1"></a>
+
+A service to white-list external id properties based on a predefined blacklist, and the amount of Schema.org definitions found in a sample external resource.
+
+This service takes in a list of  manually blacklisted external ids, and retrieves a list of currently available external ids from Wikidata.
+
+The output for this service is a listed of string representation of white-listed external ids. For example:  
+
+  ```json
+  ['P1234', 'P1233', ...]
+  ```
+
+## SS2: Fetch current mappings between Wikidata Properties and Schema.org Properties<a name="ss-2"></a>
+
+A service to retrieve the most recent state of mappings between Wikidata Properties and Schema.org properties.
+
+Outputs a list of objects representing a mapping. Each object has the following structure:
+
+* `property`: String representing a property on Wikidata.
+* `url`: A Schema.org property URL
+
+Example:
+
+```json
+[
+    {
+        "property": "P1476",
+        "url": "http://schema.org/name"
+    },
+    ...
+]
+```
+
+
+
+## SS3: Normalize data from various scraped raw formats<a name="ss-3"></a>
+
+This service takes in raw scraped data in either `json-ld`, `microdata` or `rdfa` format (For example, the results of the following scraping library: https://github.com/scrapinghub/extruct).
+
+The output of this service will be a list of objects representing a Schema.org type with the following structure:
+
+* `type`: Schema.org type url
+* `properties`: An object representing Schema.org property value pairs where the keys are Schema.org property urls, and the values are the actual data from the site. 
+
+Example:
+
+```json
+[
+    {
+        "type": "http://schema.org/Person",
+        "properties": {
+            "http://schema.org/name": "Ludwig Wittgenstein",
+            "http://schema.org/sameAs": "http://viaf.org/viaf/24609378",
+            "http://schema.org/additionalName": "Ludvigs Vitgenšteins"
+        }
+    },
+    ...
+]
+```
+
+## SS4: Map External ID Properties and Values to URL and reference meta-data<a name="ss-4"></a>
+
+This service takes in a string representation of an external id property and attempts to output a formatted url  for an external resource, as well as reference metadata according to the wikidata mapping. If non is found it will return `false`.
+
+The output format is an object with the following properties:
+
+* `url`: The formatted URL of an external resource
+
+* `referenceMetadata`: An object representing meta data for a potential reference, with the following key-value pairs:
+
+  | Key                              | Value                                     |
+  | -------------------------------- | ----------------------------------------- |
+  | "P248" ("stated in" Property id) | Wikidata item representing an external ID |
```

Comment:
"external ID" is being used inconsistently, like the line below it's "external id". I'd go with the latter.

Reply:
Will do

Reply:
Also fixed 'ID' in title to 'Id' to try and preserve title case